pages/index.py

```diff
@@ -61,6 +61,29 @@ def slider_item(title, text, active=""):
 <!--HOME SECTION START-->
 <section id="home">
 <div class="container">
+<div class="row">
+<!--DONATE SUBSECTION -->
+<div class="col-sm-12 col-md-12 col-lg-12">
+        <div class="item active donate" id="donate_button">
+             <p>$donate_text</p>
+             <div class="meter blue" id="donate_bar_small">
+                 <div class="quantity" style="left: 15px;">$donation_current_text</div>
```

Comment:
This should be `$MONEYBALANCE` - it's what's currently used for the substitution on the donation page.